modules/foreman_auth_source_ldap.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2018 Markus Bucher (ATIX AG)
```

Comment:
One other finding... This should be your name.

Reply:
Fixed.